hubblestack/returners/splunk_nova_return.py

```diff
@@ -169,6 +169,11 @@ def _generate_event(args, cloud_details, custom_fields, check_type=None, data=No
             for key, value in data[args['check_id']].items():
                 if key not in ['tag']:
                     event[key] = value
+    if check_type == 'Failure':
+        if 'detail' in data:
```

Comment:
can we make this `if check_type == 'Failure' and 'detail' in data`?

Reply:
I have pushed this change out.